src/lib.rs

```diff
@@ -194,10 +194,12 @@ mod tests {
                     policy_path
                         .with_extension("cil")
                         .file_name()
-                        .expect(&format!(
-                            "failed to extract file name from `{}`",
-                            policy_path.to_string_lossy()
-                        )),
+                        .unwrap_or_else(|| {
```

Comment:
Indeed, the --fix command includes this but there is no trace in the printed warnings… The nightly Clippy (0.1.58) adds some more fixes if you want to take them.

Reply:
Yep. I did most of the fixes by hand and carefully arranged them into groups because I wanted to make this series reviewable by humans.

I got lazy near the end seeing there was only one file remaining, so I did `cargo clippy --fix` instead for `lib.rs`.

If I have more time today, I may be able to have look at warnings from Clippy 0.1.58.